cartographer_ros/cartographer_ros/node.cc

```diff
@@ -182,25 +100,19 @@ void Node::Initialize() {
       &Node::PublishTrajectoryStates, this));
 }
 
+void Node::SpinForever() { ::ros::spin(); }
```

Comment:
Should this be renamed `Spin()` for consistency and also the fact that it will return when the node terminates?